conda.recipe/meta.yaml

```diff
@@ -13,21 +13,13 @@ build:
 
 requirements:
   build:
-    - mistune !=0.7.2
-    - nb_config_manager
     - nodejs
-    - notebook <5
+    - notebook ==4.2.0b1
```

Comment:
this could be notebook >4.1... so you don't have to change it again when the official 4.2 is out...